rflx/pyrflx/pyrflx.py

```diff
@@ -33,6 +26,21 @@ def __init__(
                     m, model.refinements, skip_message_verification
                 )
 
+    @classmethod
+    def parse(
+        cls,
+        files: List[str],
+        skip_model_verification: bool = False,
+        skip_message_verification: bool = False,
+    ) -> "PyRFLX":
```

Comment:
`parse` doesn't sound as if a new `PyRFLX` object is created. I think `from_specs` or something similar would fit better.

Reply:
Fixed by https://github.com/Componolit/RecordFlux/pull/468/commits/f4d5340c5ac47e139227bf5e76a50f4f7f681d29.